### PR TITLE
Add GCR to implementations

### PR DESCRIPTION
**Pull request by KeisukeYamashita**
## What

As title. Add Google Cloud Registry(GCR) to `implementations.md`.
If I am misunderstanding, please close this PR 🙇 

## Why

Because it was missing. 

Signed-off-by: KeisukeYamashita <19yamashita15@gmail.com>

### CODE REVIEW

File: implementations.md [hunk 1]
```diff
@@ -17,5 +17,6 @@ Projects or Companies currently adopting the OCI Image Specification
 * [coreos/rkt](https://github.com/coreos/rkt)
 * [box-builder/box](https://github.com/box-builder/box)
 * [coolljt0725/docker2oci](https://github.com/coolljt0725/docker2oci)
+* [Google Container Registry(GCR)](https://cloud.google.com/container-registry/docs/image-formats)
```
Comment (splitt3r) [inline implementations.md:20]:
```suggestion
* [Google Container Registry (GCR)](https://cloud.google.com/container-registry/docs/image-formats)
```

**Review by splitt3r — CHANGES_REQUESTED** [review]

File: implementations.md [hunk 2]
```diff
@@ -17,5 +17,6 @@ Projects or Companies currently adopting the OCI Image Specification
 * [coreos/rkt](https://github.com/coreos/rkt)
 * [box-builder/box](https://github.com/box-builder/box)
 * [coolljt0725/docker2oci](https://github.com/coolljt0725/docker2oci)
+* [Google Container Registry (GCR)](https://cloud.google.com/container-registry/docs/image-formats)
```
Comment (vbatts) [inline implementations.md:20]: @jonjohnsonjr is this correct

**Comment by jonboulle** [comment]
@KeisukeYamashita you may need to squash your branch to fix the DCO sign off

File: implementations.md (hunk 2 above)
Comment (jonjohnsonjr) [reply implementations.md:20]: Yep, should also apply to [Artifact Registry](https://pkg.dev).

**Comment by vbatts** [comment]
superceding with https://github.com/opencontainers/image-spec/pull/905

**Pull request closed without merging**